Tests/ML/pipelines/test_inference.py

```diff
@@ -198,3 +207,123 @@ def shrink_dim(i: int) -> int:
 
     def get_all_child_layers(self) -> List[torch.nn.Module]:
         return list()
+
+
+def create_config_from_dataset(input_list: List, train: List, val: List, test: List) -> DummyModel:
```

Comment:
please complete the type annotation, it should also state what the list elements are